modules/chapter2/pages/index.adoc

```diff
@@ -1,3 +1,5 @@
-= Chapter 2
+= Working with Data Science Projects
 
-This is the home page of _Chapter 2_ in the *hello* quick course....
\ No newline at end of file
+Goals:
```

Comment:
Just like the chapter 1, add one or two lines about overall chapter topics and what students will learn in this chapter